.gitignore

```diff
@@ -205,6 +205,9 @@ node_modules/
 /ajax/libs/slim.js/2.[0-5].*/
 /ajax/libs/europa/4.0.0-alpha*/
 
+
+/ajax/libs/angular-mocks/*-build*/
+
```

Comment:
@sufuf3 I add the blank lines for avoiding any conflicts because @PeterDaveHello told me that we can do that for conflicts. @PeterDaveHello Could you please help me confirm is these blank line alright? Thank you!